sites/default/statement.inc.php

```diff
@@ -21,18 +21,63 @@
  * @copyright Copyright (c) 2020 Stephen Waite <stephen.waite@cmsvt.com>
  * @author Daniel Pflieger <daniel@growlingflea.com>
  * @copyright Copyright (c) 2018 Daniel Pflieger <daniel@growlingflea.com>
+ * @author Sherwin Gaddis <sherwingaddis@gmail.com> (twig conversion)
  * @link https://github.com/openemr/openemr/tree/master
  * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
 use OpenEMR\Common\Crypto\CryptoGen;
+use OpenEMR\Common\Twig\TwigContainer;
+use Twig\TwigFilter;
+
+function pad_filter($string, $length, $pad_string = ' ', $pad_type = 'right')
+{
+    $padTypeConstant = STR_PAD_RIGHT; // Default to right padding
+
+    if (strtolower($pad_type) === 'left') {
+        $padTypeConstant = STR_PAD_LEFT;
+    } elseif (strtolower($pad_type) === 'both') {
+        $padTypeConstant = STR_PAD_BOTH;
+    }
+
+    return str_pad($string, $length, $pad_string, $padTypeConstant);
+}
+
+function match_filter($subject, $pattern)
+{
+    return preg_match($pattern, $subject);
+}
+
+function oe_format_short_date_filter($date)
```

Comment:
might want to use https://github.com/openemr/openemr/blob/master/src/Common/Twig/TwigExtension.php#L346

Reply:
Thanks @stephenwaite I will